<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTextEdit" name="ShowtextEdit">
     <property name="minimumSize">
      <size>
       <width>371</width>
       <height>161</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>371</width>
       <height>161</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTextEdit" name="sendMsgTextEdit">
     <property name="minimumSize">
      <size>
       <width>371</width>
       <height>91</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>371</width>
       <height>91</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>161</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>161</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>连接服务器</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>161</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>161</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
